src/ImageProcessor.Web.Plugins.PostProcessor/PostProcessor.cs

```diff
@@ -128,25 +128,22 @@ public static async Task<MemoryStream> PostProcessImageAsync(HttpContext context
             finally
             {
                 // Always cleanup files
-                try
+                foreach (var fileName in new[] { sourceFile, destinationFile }.Where(f => f != null))
```

Comment:
The iterator might have some overhead, but this requires less duplicated code 😉

Reply:
I've decided better performance over duplicated code, so this is converted back (although it still deletes the destination file if an exception occurred while deleting the source file first).